src/Components/Support/Cron.php

```diff
@@ -16,9 +16,17 @@ class Cron extends BladeComponent
     /** @var bool */
     public $human = false;
 
-    public function __construct(string $schedule, bool $human = false)
+    /** @var string */
+    public $locale = 'en';
+
+    /** @var bool */
+    public $use24hour = false;
+
+    public function __construct(string $schedule, bool $human = false, string $locale = 'en', bool $use24hour = false)
```

Comment:
Maybe `$is24Hour` is closer to convention?

```suggestion
    public function __construct(string $schedule, bool $human = false, string $locale = 'en', bool $is24Hour = false)
```